prediction/run.py

```diff
@@ -164,6 +175,17 @@ def _evaluate(y_gold: list, y_pred: list, is_train: bool=True) -> None:
         print(f"f1 score:  {f1:.4f}")
         print(f"accuracy:  {acc:.4f}")
 
+        cm = metrics.confusion_matrix(golds, preds)
+        unique = len(set(y_gold))
```

Comment:
Don't want to be too picky but I'd propose naming it cnt_unique (since it is count, not - unique values)